apps/web/src/services/asset-inventory/collector/modules/CollectorContents.vue

```diff
@@ -16,43 +20,42 @@
                     icon-left="ic_plus_bold"
                     @click="handleCreate"
                 >
-                    Create
+                    {{ $t('INVENTORY.COLLECTOR.MAIN.CREATE') }}
                 </p-button>
             </template>
         </p-toolbox>
-        <p-data-loader
-            class="collector-lists"
-            :data="cloudCollectorPageState.filteredList"
-            :loading="false"
-        >
-            <p-card
-                v-for="item in cloudCollectorPageState.filteredList"
-                :key="item.collector_id"
-                :header="item.name"
-                style-type="white"
+        <div class="collector-list-wrapper">
+            <div
```

Comment:
I originally implemented it using 'p-data-loader', but based on the review suggesting that it would be better to change it to 'div' due to design system dependencies, I made the modification!

Reply:
Would it be better to use 'p-data-loader' again and modify it along with other code to reduce dependencies?

Reply:
Oh it seems a miss communication... T.T
The intend of my original comment is that I recommended to use div tag inside of the p-data-loader component!
I'm so sorry that I confused you. 😭